edx_proctoring/backends/__init__.py

```diff
@@ -9,6 +9,10 @@ def get_backend_provider(exam=None):
     Returns an instance of the configured backend provider
     """
     backend_name = None
-    if exam and exam['backend']:
-        backend_name = exam['backend']
+    if exam:
+        if exam['backend']:
+            backend_name = exam['backend']
+        elif 'is_proctored' in exam and not exam['is_proctored']:
```

Comment:
This triggers only if there's an explicit 'is_proctored':False item in the exam.  Is that what you wanted?  A missing 'is_proctored' key will continue on.

Reply:
yes, that's what I intended, because there's a case where a caller calls:
`get_backend_provider({'backend': 'software_secure'})`